From f8f01cc6a82e88fa1c6d8e5d672afd4843ddcadf Mon Sep 17 00:00:00 2001
From: William Lai <b04597@freescale.com>
Date: Fri, 19 Mar 2010 16:11:05 +0800
Subject: [PATCH] ENGR00121677-2 MX53 PWM: Enabled by default in the config

Backlight based on PWM support on MX53 EVK enabled by default.

Signed-off-by: William Lai <b04597@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   48 +++++++++-----------------------------
 1 files changed, 12 insertions(+), 36 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index 737cd08..16eb284 100644
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,8 +1,10 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.31
+# Fri Mar 19 16:09:17 2010
 #
 CONFIG_ARM=y
+CONFIG_HAVE_PWM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
 CONFIG_GENERIC_GPIO=y
 CONFIG_GENERIC_TIME=y
@@ -145,6 +147,7 @@ CONFIG_FREEZER=y
 # CONFIG_ARCH_FOOTBRIDGE is not set
 CONFIG_ARCH_MXC=y
 # CONFIG_ARCH_STMP3XXX is not set
+# CONFIG_ARCH_MXS is not set
 # CONFIG_ARCH_NETX is not set
 # CONFIG_ARCH_H720X is not set
 # CONFIG_ARCH_IOP13XX is not set
@@ -192,19 +195,20 @@ CONFIG_MXC_SDMA_API=y
 CONFIG_SDMA_IRAM=y
 CONFIG_FORCE_MAX_ZONEORDER=13
 CONFIG_ARCH_MXC_HAS_NFC_V3=y
-
-#
-# MX5 Options
-#
-# CONFIG_ARCH_MX51 is not set
 CONFIG_ARCH_MX53=y
 CONFIG_MX5_OPTIONS=y
+# CONFIG_MACH_MX51_3DS is not set
+# CONFIG_MACH_MX51_BABBAGE is not set
 CONFIG_MACH_MX53_EVK=y
+
+#
+# MX5x Options:
+#
 CONFIG_ARCH_MXC_HAS_NFC_V3_2=y
 CONFIG_MXC_TZIC=y
 CONFIG_ISP1504_MXC=y
 # CONFIG_MXC_IRQ_PRIOR is not set
-# CONFIG_MXC_PWM is not set
+CONFIG_MXC_PWM=y
 CONFIG_MXC_DVFS_PER=y
 
 #
@@ -1287,6 +1291,7 @@ CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_GENERIC=y
+CONFIG_BACKLIGHT_PWM=y
 CONFIG_BACKLIGHT_MXC=y
 CONFIG_BACKLIGHT_MXC_MC13892=y
 
@@ -1446,7 +1451,6 @@ CONFIG_USB_EHCI_ROOT_HUB_TT=y
 # CONFIG_USB_SL811_HCD is not set
 # CONFIG_USB_R8A66597_HCD is not set
 # CONFIG_USB_HWA_HCD is not set
-# CONFIG_USB_GADGET_MUSB_HDRC is not set
 
 #
 # USB Device Class drivers
@@ -1514,34 +1518,6 @@ CONFIG_USB_STORAGE=y
 # CONFIG_USB_ISIGHTFW is not set
 # CONFIG_USB_VST is not set
 # CONFIG_USB_GADGET is not set
-# CONFIG_USB_GADGET_AT91 is not set
-# CONFIG_USB_GADGET_ATMEL_USBA is not set
-# CONFIG_USB_GADGET_FSL_USB2 is not set
-# CONFIG_USB_GADGET_LH7A40X is not set
-# CONFIG_USB_GADGET_OMAP is not set
-# CONFIG_USB_GADGET_PXA25X is not set
-# CONFIG_USB_GADGET_PXA27X is not set
-# CONFIG_USB_GADGET_S3C_HSOTG is not set
-# CONFIG_USB_GADGET_IMX is not set
-# CONFIG_USB_GADGET_S3C2410 is not set
-# CONFIG_USB_GADGET_M66592 is not set
-# CONFIG_USB_GADGET_AMD5536UDC is not set
-# CONFIG_USB_GADGET_FSL_QE is not set
-# CONFIG_USB_GADGET_CI13XXX is not set
-# CONFIG_USB_GADGET_NET2280 is not set
-# CONFIG_USB_GADGET_GOKU is not set
-# CONFIG_USB_GADGET_ARC is not set
-# CONFIG_USB_GADGET_LANGWELL is not set
-# CONFIG_USB_GADGET_DUMMY_HCD is not set
-# CONFIG_USB_ZERO is not set
-# CONFIG_USB_AUDIO is not set
-# CONFIG_USB_ETH is not set
-# CONFIG_USB_GADGETFS is not set
-# CONFIG_USB_FILE_STORAGE is not set
-# CONFIG_USB_G_SERIAL is not set
-# CONFIG_USB_MIDI_GADGET is not set
-# CONFIG_USB_G_PRINTER is not set
-# CONFIG_USB_CDC_COMPOSITE is not set
 
 #
 # OTG and related infrastructure
@@ -1566,7 +1542,6 @@ CONFIG_MMC_BLOCK_BOUNCE=y
 #
 # CONFIG_MMC_SDHCI is not set
 # CONFIG_MMC_MXC is not set
-# CONFIG_MMC_SPI is not set
 CONFIG_MMC_IMX_ESDHCI=y
 # CONFIG_MMC_IMX_ESDHCI_PIO_MODE is not set
 # CONFIG_MEMSTICK is not set
@@ -1583,6 +1558,7 @@ CONFIG_LEDS_MC13892=y
 # CONFIG_LEDS_LP3944 is not set
 # CONFIG_LEDS_PCA955X is not set
 # CONFIG_LEDS_DAC124S085 is not set
+# CONFIG_LEDS_PWM is not set
 # CONFIG_LEDS_BD2802 is not set
 
 #
-- 
1.5.4.4

